universal-login-react/src/ui/TopUp/Fiat/TopUpWithFiat.tsx

```diff
@@ -78,11 +80,17 @@ export const TopUpWithFiat = ({setHeaderVisible, walletService, modalClassName,
         crypto="eth"
       />;
     case 'wait':
-      return <WaitingForOnRampProvider
-        onRampProviderName={TopUpProvider.RAMP}
-        className={modalClassName}
-        logoColor={logoColor}
-      />;
+      return walletService.isKind('Deployed')
```

Comment:
it should not be able to be here if wallet is Deployed.  It should redirect to /wallet ||  /creationSuccess (at least for wallet)

Reply:
It is possible because we have also regular TopUp in dashboard